presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -231,6 +231,8 @@ Property Name                                        Description
 
 ``hive.metastore.glue.default-warehouse-dir``        Hive Glue metastore default warehouse directory
 
+``hive.metastore.glue.aws-access-key``               AWS access key to use to connect to the Glue Catalog.
```

Comment:
Also secret key :-)

Reply:
Thanks. I thought presto does not have secret key yet.